src/main/cpp/subsystems/ValorSwerve.cpp

```diff
@@ -174,3 +177,28 @@ void ValorSwerve<AzimuthMotor, DriveMotor>::setDriveClosedLoop(double mps)
 {
     driveMotor->setSpeed(mps);
 }
+
+template<class AzimuthMotor, class DriveMotor>
+void ValorSwerve<AzimuthMotor, DriveMotor>::InitSendable(wpi::SendableBuilder& builder)
+{
+    builder.SetSmartDashboardType("Susbsystem");
+    builder.AddDoubleProperty(
+        "magEncoderRotatons",
+        [this] { return getMagEncoderCount() / MAG_ENCODER_TICKS_PER_REV; },
+        nullptr
+    );
+    // @todo AddRawProperty must take in a byte[]. Figure out how to convert from SwerveModulePosition & SwerveModuleState. add AddRawProperty in markdown file
+

```

Comment:
Instead of using RawProperty, can we use DoubleProperty and just take individual components out of the struct?

Reply:
nevermind. That would not allow us to store the names of the different variables within them with the values